packages/nextra/tsup.config.ts

```diff
@@ -14,6 +14,8 @@ const CLIENT_ENTRY = [
 const entries = fg.sync(CLIENT_ENTRY, { absolute: true })
 const entriesSet = new Set(entries)
 
+const winPath = (p: string) => p.replace(/\\/g, '/')
```

Comment:
Could you use `slash` package for this case (he is already installed in nextra)

Reply:
> Could you use `slash` package for this case (he is already installed in nextra)

Edited